spec/requests/portfolios_spec.rb

```diff
@@ -0,0 +1,135 @@
+describe 'Portfolios API' do
+  include RequestSpecHelper
+
+  let!(:portfolio)            { create(:portfolio) }
+  let!(:portfolio_item)       { create(:portfolio_item) }
+  let!(:portfolio_items)      { create_list(:portfolio_item, 3, :portfolios => [portfolio]) }
+  let(:portfolio_id)          { portfolio.id }
+  let(:portfolio_item_id)     { portfolio_items.first.id }
+  let(:new_portfolio_item_id) { portfolio_item.id }
+  let(:tenant)                { create(:tenant, :with_external_tenant) }
+
+  # Encoded Header: { 'identity' => { 'is_org_admin' => false, 'org_id' => 111 } }
+  let(:user_encode_key_with_tenant) { { 'x-rh-auth-identity' => 'eyJpZGVudGl0eSI6eyJpc19vcmdfYWRtaW4iOmZhbHNlLCJvcmdfaWQiOiIxMTEifX0=' } }
+  # Encoded Header: { 'identity' => { 'is_org_admin' => true, 'org_id' => 111 } }
+  let(:admin_encode_key_with_tenant) { { 'x-rh-auth-identity' => 'eyJpZGVudGl0eSI6eyJpc19vcmdfYWRtaW4iOnRydWUsIm9yZ19pZCI6MTExfX0=' } }
+
+  %w(admin user).each do |tag|
+    describe "GET #{tag} tagged /portfolios/:portfolio_id" do
+      before do
+        get "#{api}/portfolios/#{portfolio_id}", :headers => send("#{tag}_encode_key_with_tenant")
+      end
+
+      context 'when portfolios exist' do
+        it 'returns status code 200' do
+          expect(response).to have_http_status(200)
+        end
+
+        it 'returns all portfolio requests' do
+          expect(json).not_to be_empty
+          expect(json['id']).to eq(portfolio_id)
+        end
+      end
+    end
+
+    describe "GET #{tag} tagged /portfolios/:portfolio_id/portfolio_items" do
+      before do
+        get "#{api}/portfolios/#{portfolio_id}/portfolio_items", :headers => send("#{tag}_encode_key_with_tenant")
```

Comment:
@syncrou we are trying to build this out
https://github.com/ManageIQ/service_portal-api/blob/ccca63a2289b43874e280f6c95ff9c25e98d8281/spec/support/service_spec_helper.rb#L6

If you want you can add all of these headers into the helper method

Reply:
@syncrou I am not sure if you do this to check coverage locally

In the terminal
```
export CI=1
rspec <<your spec file>>
open coverage/index.html
```

You can see the coverage for each file and for the spec that you ran.